packages/jest-util/src/index.ts

```diff
@@ -28,3 +28,6 @@ export {default as pluralize} from './pluralize';
 export {default as formatTime} from './formatTime';
 export {default as tryRealpath} from './tryRealpath';
 export {default as requireOrImportModule} from './requireOrImportModule';
+export {default as shuffleArray, rngBuilder} from './shuffleArray';
```

Comment:
Could we move this into `jest-circus` directly rather than having it in `jest-util`? Not used outside of it, and I don't think we want to maintain a public API for it